site/_includes/example.html

```
@@ -3,34 +3,57 @@
 </div>
 {:/nomarkdown}
 
+<!-- Visual example snippet -->
+{% capture visual-example %}
 <div class="layout__section">
-	{% if include.content %}
-		<div class="layout__section layout__section--fitted layout__section--spaced-80 layout__section--bordered layout__section--rounded">
-			{%- if include.hide_preview == null -%}
-				<div class="layout__section">
-					<div class="layout__inner layout__inner--padded-90">
-						<div>
-							{{- include.content -}}
+	<div class="layout__inner layout__inner--padded-90">
+		<div>
+			{{- include.content -}}
+		</div>
+	</div>
+</div>
+{% endcapture %}
+
+<!-- Code example snippet -->
+{% capture code-example %}
+	{%- highlight html -%}
+		{{- include.content -}}
+	{%- endhighlight -%}
+{% endcapture %}
+
+<div class="layout__section layout__section--spaced-70 layout__section--fitted layout__section--bordered layout__section--rounded">
+
+	<!-- Default example including visual example and code example -->
+	{%- if include.show-code-example-only == null and include.show-visual-example-only == null -%}
```

Not sure if I like the show-only syntax. When you display everything by default, it feels more logic to hide something.